WalletWasabi/WabiSabi/Crypto/UnboundedChannel.cs

```diff
@@ -0,0 +1,37 @@
+using System.Threading;
+using System.Threading.Tasks;
+using System.Collections.Concurrent;
+
+namespace WalletWasabi.WabiSabi.Crypto
+{
+	public class UnboundedChannel<T>
```

Comment:
Is this relevant? https://docs.microsoft.com/en-us/dotnet/api/system.threading.channels.channel?view=net-5.0